### PR TITLE
fix(components): Fix input group spacing issues

### PR DESCRIPTION
**Pull request by dlrussel**
## Motivations

Due to the fact that FormField wrapper was using a fragment when a form field had a description or a validation message these we being inputted into the DOM as siblings of the wrapper with no parent element to contain them. 

This caused layout issues when input groups were used with the horizontal variation and description/validation errors were being placed into the flex container as siblings of the Form Field Wrapper and being displayed in a row horizontally. See image below:
image

This also created spacing inconsistencies for validations/descriptions when inputs were used inside of a `<Content/>` component vs standalone because the content component styles all direct children with padding and therefore description and validation were also having bottom padding applied to them instead of just the group.

## Changes

To resolve this I have added a div that will act as a container element that can wrap Form Field Wrapper as well as description and validation messages so that these elements can be styled as a group. 
See image below:
image

### Fixed

- I also updated the invalid class so that it is position relative so that the z-index issue we were seeing with the invalid border color displays properly.
Before:
image

After:
image

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Review by darryltec — CHANGES_REQUESTED** [review]
Code looks good to me. Snapshots were easy to check after turning off whitespace changes.

Did saw a bit of an issue with our inline input pattern tho

Master
image

This branch
image


[codesandbox ](https://codesandbox.io/s/white-tree-kj5fhw?file=/package.json)

_If only we have chromatic. One day!_

File: packages/components/src/FormField/FormField.css [hunk 1]
```diff
@@ -113,8 +118,9 @@
 }
 
 .inline {
-  display: inline-flex;
+  display: inline-block;
```
[the remark this answers is not in the document]
Comment (dlrussel) [reply packages/components/src/FormField/FormField.css:121]: yes!

File: packages/components/src/FormField/FormField.css [hunk 2]
```diff
@@ -113,8 +118,9 @@
...
   width: auto;
+  margin-left: var(--space-base);
```
[the remark this answers is not in the document]
Comment (dlrussel) [reply packages/components/src/FormField/FormField.css]: ah ok good context I have removed this and we can revisit it later on if we want to!

**Review by darryltec — APPROVED** [review]
Alright! We good!! Also fixed the form issue

This branch
image

Master
image

_Perfecto_

**Pull request merged**